install_builder/deadline-cloud-for-cinema-4d.xml

```diff
@@ -0,0 +1,132 @@
+                    <onPackingFilterList>
+                        <fileNameFilter pattern="*/deadline/cinema4d_submitter/*" logic="matches" patternType="glob"/>
+                    </onPackingFilterList>
+                </distributionDirectory>
+            </distributionFileList>
+        </folder>
+    </folderList>
+    <initializationActionList>
+        <if>
+            <conditionRuleList>
+                <compareText>
+                    <logic>does_not_contain</logic>
+                    <text>${platform_name}</text>
+                    <value>linux</value>
+                </compareText>
+            </conditionRuleList>
+            <actionList>
+                <setInstallerVariable name="all_components" value="${all_components} deadline_cloud_for_cinema_4d" />
+            </actionList>
+            <elseActionList>
+                <setInstallerVariable name="component(deadline_cloud_for_cinema_4d).show" value="0"/>
+            </elseActionList>
+        </if>
+        <setInstallerVariable name="cinema_4d_submitter_installdir" value="${installdir}\Submitters\Cinema4D" />
+        <if>
+            <conditionRuleList>
+                <platformTest type="windows" />
+            </conditionRuleList>
+            <actionList>
+                <setInstallerVariable name="cinema_4d_deps_platform" value="windows" />
+            </actionList>
+        </if>
+    </initializationActionList>
+    <parameterList>
+        <stringParameter name="deadline_cloud_for_cinema_4d_summary" ask="0" cliOptionShow="0">
+            <value>Deadline Cloud for Cinema 4D
+- Register the extension with Cinema 4D by moving the DeadlineCloud.pyp file to the Cinema 4D plugins folder. 
+- Set the C4DPYTHONPATH311 to retrieve deadline modules. 
+            </value>
+        </stringParameter>
+        <directoryParameter>
+            <name>cinema_4d_installdir</name>
+            <title>Cinema 4D</title>
+            <description>Cinema 4D Installation Directory</description>
+            <explanation>Path to Cinema 4D installation directory. For easiest installation, Cinema 4D should be installed before installing Deadline Cloud for Cinema 4D.</explanation>
+            <allowEmptyValue>0</allowEmptyValue>
+            <ask>yes</ask>
+            <cliOptionName>cinema4d-installation-directory</cliOptionName>
+            <cliOptionText>Path to Cinema 4D installation directory. </cliOptionText>
+            <mustBeWritable>yes</mustBeWritable>
+            <mustExist>1</mustExist>
+        </directoryParameter>
+    </parameterList>
+    <postInstallationActionList>
+        <createDirectory>
+            <path>${cinema_4d_installdir}/plugins</path>
+            <ruleList>
+                <fileTest condition="not_exists" path="${cinema_4d_installdir}/plugins"/>
+            </ruleList>
+        </createDirectory>
+        <copyFile>
+            <destination>${cinema_4d_installdir}/plugins/DeadlineCloud.pyp</destination>
```

Comment:
Does this get removed on uninstall? In the past I've seen files moved by `copyFile` not get removed by default on uninstall.

Reply:
It does not, thanks I've updated it now, I'll test now.